ironic.conf

```diff
@@ -5,12 +5,14 @@ default_boot_interface = ipxe
 default_deploy_interface = direct
 default_inspect_interface = inspector
 default_network_interface = noop
+default_raid_interface = no-raid
```

Comment:
This is not overly useful and will complicate using RAID. I suggest dropping it.

Reply:
Done. I removed this config